src/strings-in-block/class-html.php

```diff
@@ -162,12 +164,15 @@ private function get_block_queries( \WP_Block_Parser_Block $block ) {
 	/**
 	 * @param \WP_Block_Parser_Block $block
 	 *
-	 * @return ListBlock|StandardBlock
+	 * @return ListBlock|StandardBlock|HtmlBlock
 	 */
 	private function get_dom_handler( \WP_Block_Parser_Block $block ) {
 		if ( self::LIST_BLOCK_NAME === $block->blockName ) {
 			return new ListBlock();
 		}
+		if ( self::HTML_BLOCK_NAME === $block->blockName ) {
+			return new HtmlBlock();
+		}
```

Comment:
@mrudzki91, I suggest this smart pattern to replace kind of switch:

```php
$class = wpml_collect( [
    self::LIST_BLOCK_NAME => ListBlock::class,
    self::HTML_BLOCK_NAME => HtmlBlock::class,
] )->get( $block->blockName, StandardBlock::class );

return new $class();
```

Credits: @brucepearson :)

Reply:
@strategio great idea, thanks! :)